presto-docs/src/main/sphinx/connector/hive-security.rst

```diff
@@ -404,13 +406,43 @@ Keytab files must be distributed to every node in the cluster that runs Presto.
 
 :ref:`Additional Information About Keytab Files.<hive-security-additional-keytab>`
 
+.. _hive-security-metastore-impersonation:
+
+Impersonation Accessing the Hive Metastore
+------------------------------------------
+
+Presto supports impersonating the end user when accessing the Hive metastore.
+Metastore impersonation can be enabled with
+
+.. code-block:: none
+
+    hive.metastore.thrift.impersonation.enabled=true
+
+When using ``KERBEROS`` Metastore authentication with impersonation, the principal
+specified by the ``hive.metastore.client.principal`` property must be allowed to
+impersonate the current Presto user, as discussed in the section
+:ref:`configuring-hadoop-impersonation`.
+
+The impersonation is applied when:
+
```

Comment:
I would rather see specific use cases when Metastore impersonation should be used (e.g storage based authorization) instead of listing de-facto Metastore API call groups

Reply:
@sopel39 this might be a good idea. Can you log an issue for this?